### PR TITLE
Run flake8 on CI

### PR DESCRIPTION
**Pull request by nre-ableton**
- Add docstrings to tests
- Add flake8 to Python requirements
- Run flake8 on CI

### CODE REVIEW

**Comment by nre-ableton** [comment]
@ablbot autosquash

File: molecule/default/tests/test_default.py [hunk 1]
```diff
@@ -1,14 +1,16 @@
+"""Molecule tests for the default scenario."""
+
 import os
 
 import testinfra.utils.ansible_runner
 
-
```
Comment (ala-ableton) [inline molecule/default/tests/test_default.py:5]: This line should not be removed in 9e99bf19cd56e24359267475ca6936a7d2f2cee7. I guess we need some isort configuration.

**Review by ala-ableton — CHANGES_REQUESTED** [review]

**Review by ala-ableton — COMMENTED** [review]
@nre-ableton I pushed a fixup commit to improve the isort configuration. Please squash e69f5d2 into 9e99bf1, thanks!

**Comment by nre-ableton** [comment]
@ablbot autosquash

**Comment by ala-ableton** [comment]
@nre-ableton actually, `include_trailing_comma = 1` is part of `profile = black` (see https://pycqa.github.io/isort/docs/configuration/profiles.html), so we can remove it.

**Comment by nre-ableton** [comment]
@ablbot autosquash

**Review by ala-ableton — CHANGES_REQUESTED** [review]
Please squash e69f5d2 into 9e99bf1

**Review by ala-ableton — APPROVED** [review]

**Pull request merged**